core/src/main/scala/org/apache/spark/sql/tispark/TiRowRDD.scala

```diff
@@ -41,6 +38,59 @@ class TiRowRDD(override val dagRequest: TiDAGRequest,
                @transient private val sparkSession: SparkSession)
     extends TiRDD(dagRequest, physicalId, tiConf, tableRef, session, sparkSession) {
 
+  protected val logger: Logger = log
+
+  def allowTiFlashRead(): Boolean =
```

Comment:
check duplicate here.

Reply:
Could you be more specific?

Reply:
You have two `allowTiFlashRead()` existing in code. I'm not sure if it is necessary.